Modules/Loadable/Segmentations/EditorEffects/Python/AbstractScriptedSegmentEditorAutoCompleteEffect.py

```diff
@@ -49,6 +49,22 @@ def __del__(self, scriptedEffect):
     self.delayedAutoUpdateTimer.stop()
     self.observeSegmentation(False)
 
+  @staticmethod
+  def isBackgroundLabelmap(labelmapOrientedImageData):
+    if labelmapOrientedImageData is None:
+      return False
+    # If five or more corner voxels of the image contain non-zero, then it is background
+    extent = labelmapOrientedImageData.GetExtent()
+    numberOfFilledCorners = 0
+    for i in [0,1]:
+      for j in [2,3]:
+        for k in [4,5]:
+          if labelmapOrientedImageData.GetScalarComponentAsFloat(extent[i],extent[j],extent[k],0) > 0:
```

Comment:
Add a check that the extent is not empty (extent[i]<=extent[i+1] along all three axes).